apps/prairielearn/src/models/assessment.ts

```diff
@@ -20,3 +20,17 @@ export async function selectOptionalAssessmentById(
 ): Promise<Assessment | null> {
   return await queryOptionalRow(sql.select_assessment_by_id, { assessment_id }, AssessmentSchema);
 }
+
+const AssessmentInfoSchema = z.object({
+  assessment_label: z.string(),
+  course_instance_id: IdSchema,
+  course_id: IdSchema,
+});
+
+export async function selectAssessmentInfoForJob(assessment_id: string) {
+  return await queryRow(
+    sql.select_assessment_info_for_job,
+    { assessment_id },
+    AssessmentInfoSchema,
+  );
+}
```

Comment:
Nit, to avoid potential ambiguities or collisions:

```suggestion
const AssessmentInfoForJobSchema = z.object({
  assessment_label: z.string(),
  course_instance_id: IdSchema,
  course_id: IdSchema,
});

export async function selectAssessmentInfoForJob(assessment_id: string) {
  return await queryRow(
    sql.select_assessment_info_for_job,
    { assessment_id },
    AssessmentInfoForJobSchema,
  );
}
```

Reply:
You could even inline the schema in the function if you don't want to have to name it.

Reply:
Inlined the schema in 8f763802afe71d05622995f183f5fe4b6e5fcc61.